config/seatunnel.yaml

```diff
@@ -25,6 +25,9 @@ seatunnel:
     print-job-metrics-info-interval: 60
     slot-service:
       dynamic-slot: true
+    coordinator-service:
+      core-thread-num: 30
+      max-thread-num: 1000
```

Comment:
when add this parameter, it will limit the thread size.
now the default is int max, if we use a small value as the default value, it maybe cause exception when user upgrade version.
so I suggest use int max as default value, and describe this feature in document let user update it by self.

Reply:
I see the default value is int max, we can remove `max-thread-num` config from yaml, If user need change, they can update it by self.

Reply:
> I see the default value is int max, we can remove `max-thread-num` config from yaml, If user need change, they can update it by self.

done.